src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -578,7 +578,7 @@ public function putAction($id, Request $request)
 
                 $view = $this->view($acc, 200);
                 $view->setSerializationContext(
-                    SerializationContext::create()->setGroups(['fullAccount', 'partialContact', 'partialMedia'])
+                    SerializationContext::create()->setGroups(['fullAccount', 'partialContact', 'partialMedia', 'partialCategory'])
```

Comment:
Don't you think that should be extracted into an own variable as in the `ContactController`?